components/brave_wallet/browser/zcash/zcash_wallet_service.cc

```diff
@@ -295,6 +251,19 @@ void ZCashWalletService::ValidateZCashAddress(
   std::move(callback).Run(mojom::ZCashAddressValidationResult::Success);
 }
 
+void ZCashWalletService::PostShieldTransaction(
+    const std::string& chain_id,
+    ZCashTransaction zcash_transaction,
+    SignAndPostTransactionCallback callback) {
+  auto tx = ZCashSerializer::SerializeRawTransaction(zcash_transaction);
+  std::string as_string(reinterpret_cast<char*>(tx.data()), tx.size());
+  zcash_rpc_->SendTransaction(
+      chain_id, as_string,
+      base::BindOnce(&ZCashWalletService::OnSendTransactionResult,
```

Comment:
`base::span<const uint8_t>` should work instead of `std::string` all the way to `MakeSendTransactionParams(const std::string& data)`

Reply:
Done